doc/whats_new/v1.3.rst

```diff
@@ -101,6 +101,10 @@ Changelog
   :pr:`24935` by :user:`Seladus <seladus>`, :user:`Guillaume Lemaitre <glemaitre>`, and
   :user:`Dea María Léon <deamarialeon>`.
 
+- |Fix| Make :class:`AdditiveChi2Sampler` stateless.
```

Comment:
Since `check_estimator` is public I think that it makes sense to add an entry in `sklearn.utils` mentioning that we added an additional check. It could be informative for third-party libraries using our checks.